db/migrate/20170117211441_create_user_votes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUserVotes < ActiveRecord::Migration[5.0]
```

Comment:
One thing at a time. Make migrations for users and items. Delete the other two until you begin working on those features. 

Additionally, in the future, you will need to actually create the feature that will be used in tandem with those migrations, but I want to get your codeship running, so I'll allow it.